Resources/Prototypes/_NF/Mail/mailDeliveries.yml

```diff
@@ -28,7 +28,7 @@
     MailCosplayArc: 0.3
     MailCosplayGeisha: 0.3
     MailNFCosplayMaid: 0.25
-    MailNFTacticalMaid: 0.05 # Shares a pool with NFCosplayMaid as a rare variant
+    MailNFTacticalMaid: 0.15 # Shares a pool with NFCosplayMaid as a rare variant
```

Comment:
CosplayMaid to 0.2,
TacticalMaid to 0.1?

I'd suggest keeping the group weight at 0.3 unless you really want more maid RP.  The cosplay set are some of the worst drops IMO, even worse than food.

Reply:
The thing is that a DAW is a lot less niche than this.  Most people like to play or hear music, so that's fine.  Not everybody wants to play Sister Invicta vs. the space lube slimes.

Within mail, I still think it's best to group like items into pools, weight them appropriately within pools, and then weight those pools against each other.  I would argue that CosplayMaid and TacticalMaid should be lumped together, and I don't think the weight of that should be much higher than, say, the nurse outfit.

Reply:
fixed on last commit.